app/models/user.rb

```diff
@@ -1,3 +1,4 @@
 class User < ApplicationRecord
   include Clearance::User
+  has_many :articles
```

Comment:
**nitpick:** Add a new line.

Everybody loves a bit of breathing space. Let your code breathe!